### ISSUE TITLE
Days of the previous month should be displayed

### ISSUE THREAD

**Issue by FezVrasta** [alienfast/picker.js#9]
![image](https://cloud.githubusercontent.com/assets/5382443/14924749/4f0e50ae-0e44-11e6-9d32-ffd9d571cbc3.png)

Maybe in gray

**Comment by rosskevin** [comment]
That's actually per the spec:
https://www.google.com/design/spec/components/pickers.html#pickers-date-pickers

It was in gray, and the spec doesn't mandate it...I can go either way.  You prefer gray?

**Comment by FezVrasta** [comment]
Oh I see... it just looks weird to not have the days of the past/next month visible to fill the entire grid... Usualy the calendars look like that

**Comment by rosskevin** [comment]
agreed, but the `visibility: none` on this is actually causing #10.  I can implement either way.

**Comment by rosskevin** [comment]
So I checked two places:
- google calendar web - it shows past/next month's days
- google calendar android - it does not show past/next month's days

The Android mobile seems more material, so I'm inclined to keep these as not visible.  I did however adjust the stylesheet for those that want it.

In the case someone wants to show past/future, they can just override `.old` and `.new` and provide `visibility: visible`.
